wiki/Tournaments/MALT/1/en.md

```diff
@@ -0,0 +1,504 @@
+
+#### Qualifier Stage
+
+- Mappools were constructed to have 5 maps, consisting of 2 Rice-based, 2 LN-based, and 1 Hybrid.
+- Players were scheduled to a multiplayer room according to their availability.
+- Each player got 2 attempts at each map, where only the best attempt for each map will count
+- Each map had its own leaderboard, and each player's overall seed was determined by the average of these ranks. The top 48 players of the qualifiers proceeded to the next rounds.
+- The top 16 players immediately advanced to the Bracket Stage. Players seeded from 17 to 48 had to participate in the play-offs.
+
+#### Play-offs
+
+- Players who were participating in this round had to face each other in a one-on-one Best of 7 match.
+- Match ups were determined by this formula: ``65 - seed``. (This meant that seed 17 faced seed 48, 18 vs 47, and so on)
+- Mappool Size was determined as follows: 4 Rice-based, 2 LN-based, 3 Hybrids, 1 Tiebreaker
+- Winners from the play-offs took the final 16 slots in the Bracket Stage. The losers were eliminated.
+
+#### Bracket Stage
+
+- Players were seeded into a double-elimination bracket. They comprise of the Top 16 from the qualifiers and the winners from the Play-offs to top up the remaining 16 slots.
+- Mappool Sizes and Best of Format for each round were given as follows:
+  - Round of 32 (Best of 7) - 4 Rice-based, 2 LN-based, 3 Hybrids, 1 Tiebreaker
+  - Round of 16 (Best of 9) - 5 Rice-based, 3 LN-based, 3 Hybrids, 1 Tiebreaker
+  - Quarterfinals and Semifinals (Best of 11)- 6 Rice-based, 4 LN-based, 3 Hybrids, 1 Tiebreaker
+  - Finals and Grand Finals (Best of 13) - 7 Rice-based, 5 LN-based, 3 Hybrids, 1 Tiebreaker
+
+### Match Rules
+
+- The multiplayer room will be created and invites sent to players 5-10 minutes prior to the match start time.
+- If a player is more than 10 minutes late to the for their match, the player inside the MP room will be declared the winner. If neither player appears, the player with the higher seed will be considered the winner.
+- Each player will be allowed to select 1 map as a warmup. Warmups must be uploaded to the osu! servers and cannot exceed 4 minutes in length.
+- Players are required to choose to ban 1 map from the mappool (excluding the tiebreaker). The pick/ban order is determined by the !roll command. The player with the lower roll bans first and the player with the higher roll picks first. Players will alternate picks until a match winner is decided.
+- If both players are tied on match point (e.g. 3-3 score in a Best-of-7), the players will play the tiebreaker map of the pool to determine the winner.
+- Players are given 2 minutes to ban/pick maps when it is their turn. Failure to do so means that the referee will ban/pick at random in behalf of the player.
+- Players are eligible to use a 2 minute tactical time out once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.
```

Comment:
```suggestion
- Players are eligible to use a 2-minute tactical timeout once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.
```